adafruit_ht16k33/ht16k33.py

```diff
@@ -40,10 +40,12 @@
 
 class HT16K33:
     """The base class for all displays. Contains common methods."""
```

Comment:
Can you add a note here that `auto_write` is on by default?

Reply:
Is that just to make it more obvious than what Read The Docs will have? Otherwise, it's similar to the default value for `address`.